airgun/helpers/host.py

```diff
@@ -3,9 +3,8 @@
 
 class HostHelper(BaseEntityHelper):
 
-    def form_create_fill_read(self, values, read_widget_names=None):
-        """Goto create host view, fill the view with supplied values, and return the view read
-        fields values, will read only read_widget_names if supplied otherwise read the fields in
-        values.
+    def read_create_view(self, values, read_widget_names=None):
+        """Goto create create host view, fill the view with supplied values, and return widgets
```

Comment:
```
Open create host view, fill entity fields with supplied values, and then return widgets values. Read values from 'read_widget_names' list if provided otherwise read values from all widgets
```